.github/workflows/daily_reports.yaml

```diff
@@ -44,20 +54,36 @@ jobs:
       - name: Setup Hive
         run: make setup-hive
 
-      - name: Run Hive Simulation
-        run: cd hive && ./hive --client ethrex --sim ${{ matrix.simulation }} --sim.parallelism 16
+      - name: Run Hive RPC Simulation
```

Comment:
Why did you start running these serially?

Reply:
It seems we want to achieve similar things, see: https://github.com/lambdaclass/ethrex/pull/1666

Reply:
There are two reasons for this:
- It was easier to debug.
- I was facing issues with rate limiting when running the tests, I figured that maybe running them serially would help work around this.

Reply:
Closing this since I went back to running them in parallel.